translator/translate/otel/pipeline/applicationsignals/translator.go

```diff
@@ -60,11 +60,12 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 	}
 
 	mode := context.CurrentContext().KubernetesMode()
+	translators.Processors.Set(resourcedetection.NewTranslator(resourcedetection.WithDataType(t.dataType)))
```

Comment:
Ahh it makes sense to move this before awsentitystore processor to make use the resource tags